lib/StateStoreError.js

```diff
@@ -14,10 +14,13 @@ const { ErrorWrapper, createUpdater } = require('@adobe/aio-lib-core-errors').Ai
 const logger = require('@adobe/aio-lib-core-logging')('@adobe/aio-lib-state', { provider: 'debug' })
 
 /**
- * @typedef StateLibError
- * @private
- *
+ * @typedef {Object} StateLibError
```

Comment:
`Object` should be `object` according to linter errors below

Reply:
fixed